cookbooks/aws-parallelcluster-environment/resources/lustre/lustre_rocky8.rb

```diff
@@ -25,13 +25,15 @@
 
 action :setup do
   version = node['platform_version']
+  log "Installing FSx for Lustre. Platform version: #{version}, kernel version: #{node['cluster']['kernel_release']}"
   if version.to_f < 8.2
     log "FSx for Lustre is not supported in this Rocky Linux version #{version}, supported versions are >= 8.2" do
       level :warn
     end
-    # rhel8 kernel 4.18.0-425.3.1.el8 has broken kABI compat https://github.com/openzfs/zfs/issues/14724
-  elsif node['cluster']['kernel_release'].include? "4.18.0-425.3.1.el8"
-    log "FSx for Lustre is not supported in kernel version 4.18.0-425.3.1.el8 of Rocky Linux, please update the kernel version" do
+  elsif version.to_f == 8.7 && (node['cluster']['kernel_release'].include?("4.18.0-425.3.1.el8") || node['cluster']['kernel_release'].include?("4.18.0-425.13.1.el8_7"))
```

Comment:
Why is `version.to_f == 8.7` needed? If it is needed, is the logical operators precedence correct?

Reply:
I will revisit the version number with Enrico